includes/Story_Renderer/Embed.php

```diff
@@ -64,34 +64,50 @@ public function __construct( Story $story ) {
 	 * @return string Rendered block type output.
 	 */
 	public function render( array $args = [] ) {
-		$defaults     = [
+		$defaults = [
 			'align'  => 'none',
 			'class'  => 'wp-block-web-stories-embed',
 			'height' => 600,
 			'width'  => 360,
 		];
-		$args         = wp_parse_args( $args, $defaults );
-		$align        = sprintf( 'align%s', $args['align'] );
-		$class        = $args['class'];
-		$url          = $this->story->get_url();
-		$title        = $this->story->get_title();
-		$poster       = ! empty( $this->story->get_poster_portrait() ) ? esc_url( $this->story->get_poster_portrait() ) : '';
-		$margin       = ( 'center' === $args['align'] ) ? 'auto' : '0';
-		$player_style = sprintf( 'width: %dpx;height: %dpx;margin: %s', absint( $args['width'] ), absint( $args['height'] ), esc_attr( $margin ) );
-		$poster_style = ! empty( $poster ) ? sprintf( '--story-player-poster: url(%s)', $poster ) : '';
 
-		ob_start();
+		$args   = wp_parse_args( $args, $defaults );
+		$align  = sprintf( 'align%s', $args['align'] );
+		$class  = $args['class'];
+		$url    = $this->story->get_url();
+		$title  = $this->story->get_title();
+		$poster = ! empty( $this->story->get_poster_portrait() ) ? esc_url( $this->story->get_poster_portrait() ) : '';
```

Comment:
```suggestion
		$poster = ! empty( $this->story->get_poster_portrait() ) ? esc_url_raw( $this->story->get_poster_portrait() ) : '';
```

Since `esc_url_raw()` is for sanitization. The escaping is done below when `$poster_style` is output inside of the `style` attribute.